tests/manual-test-cases/Group17-VIC-UI/17-1-VIC-UI-Installer.robot

```diff
@@ -0,0 +1,143 @@
+    Install Vicui Without Webserver Nor Bash  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}
+    ${output}=  OperatingSystem.GetFile  install.log
+    Run Keyword And Continue On Failure  Should Contain  ${output}  Bash shell is required
+    Force Remove Vicui Plugin
+    Remove File  install.log
+
+Install Successfully Without Webserver
+    [Timeout]  ${TIMEOUT}
+    Set Vcenter Ip
+    Install Vicui Without Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}
+    ${output}=  OperatingSystem.GetFile  install.log
+    ${run_results}=  Run Keyword And Return Status  Should Contain  ${output}  was successful
+    ${now}=  Run  date +%m%d%y-%H%M%S
+    Run Keyword If  ${run_results} == False  Copy File  install.log  install-FAIL-${now}.log
+    Remove File  install.log
+
+Attempt To Install When Plugin Is Already Registered
+    # Plugin is already installed at this moment on he target VCSA
+    # Try installing the plugin and see if it fails in an expected way
+    [Timeout]  ${TIMEOUT}
+    Set Vcenter Ip
+    Install Fails At Extension Reg  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${TRUE}
+    ${output}=  OperatingSystem.GetFile  install.log
+    Should Contain  ${output}  is already registered
+    Remove File  install.log
+
+Install Successfully Without Webserver Using Force Flag
+    # Plugin is already installed at this moment on he target VCSA
+    # Install the plugin using the --force flag and see if it succeeds
+    [Timeout]  ${TIMEOUT}
+    Set Vcenter Ip
+    Install Vicui Without Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${TRUE}
+    ${output}=  OperatingSystem.GetFile  install.log
+    ${run_results}=  Run Keyword And Return Status  Should Contain  ${output}  was successful
+    ${now}=  Run  date +%m%d%y-%H%M%S
+    Run Keyword If  ${run_results} == False  Copy File  install.log  install-FAIL-${now}.log
+    Remove File  install.log
+    Log To Console  Force removing Vicui for next tests...
+    Force Remove Vicui Plugin
+
+Attempt To Install With Webserver And Wrong Path To Plugin
+    # Try installing the plugin using a web server while providing VIC_UI_HOST_URL that does not exist and see if it fails in an expected way
+    Set Vcenter Ip
+    ${results}=  Replace String Using Regexp  ${configs}  VCENTER_IP=.*  VCENTER_IP=\"${TEST_VC_IP}\"
+    ${results}=  Replace String Using Regexp  ${results}  VIC_UI_HOST_URL=.*  VIC_UI_HOST_URL=\"http:\/\/this-fake-host\.does-not-exist\"
+    Create File  ${UI_INSTALLER_PATH}/configs  ${results}
+    Wait Until Created  ${UI_INSTALLER_PATH}/configs
+    Install Fails At Extension Reg  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${FALSE}
+    ${output}=  OperatingSystem.GetFile  install.log
+    Should Contain  ${output}  Could not resolve the host
+    Remove File  install.log
+
+Uninstall VCH
```

Comment:
Each of these is a separate test case so I think the install and uninstall VCH should be part of suite setup and tear down rather than a test case